### DISCUSSION TITLE
为什么无法用Aria2离线下载？

### DISCUSSION THREAD

**Discussion by fandouke** [AlistGo/alist#4005] [Q&A] [answered]
我开启了Aira2 然后添加离线下载，为什么会提示：
Failed get storage: please add a storage first.
具体需要在哪里设置吗？

**Comment by veenyi** [comment]
不是告诉你，失败了，请先添加存储先吗？

**Comment by wy580477** [comment, marked as answer]
> 我开启了Aira2 然后添加离线下载，为什么会提示： Failed get storage: please add a storage first. 具体需要在哪里设置吗？

需要先点进去某个存储的目录才可以，一开始也迷惑了我。。

**Comment by fandouke** [comment]
从哪里添加进去 ，我已经添加的存储了啊。。

**Comment by fandouke** [comment]
https://www.helloimg.com/images/2023/03/31/o3pWrP.png
是不是添加这个为什么我的这个地方没有下载目录，上面这个图是从BILIBILI网https://www.bilibili.com/video/BV1ie411T7oD/
截取的，下面这个是我的
https://www.helloimg.com/images/2023/03/31/o3pNQ5.png

**Reply by veenyi** [reply]
添加了存储之后，进入你的存储目录下面再用Aria2下载，相当于是在指定文件夹下面下载文件到指定的地方

**Reply by wy580477** [reply]
> https://www.helloimg.com/images/2023/03/31/o3pWrP.png 是不是添加这个为什么我的这个地方没有下载目录，上面这个图是从BILIBILI网https://www.bilibili.com/video/BV1ie411T7oD/ 截取的，下面这个是我的 https://www.helloimg.com/images/2023/03/31/o3pNQ5.png

此Aria2非彼Aria2，图里显示的是连接你本地的Aria2下载文件到本地。离线下载功能是，你当前在哪个目录，就离线下载到哪个目录。

**Comment by fandouke** [comment]
还是不懂操作，能截个图给我看看具体在哪个位置操作吗？

**Reply by anjia0532** [reply]
你先添加存储，然后进入你要下载到哪个盘的哪个目录，再点右下角的离线下载，aria2就会下载到当前所在目录。
![image](https://github.com/alist-org/alist/assets/15098916/9cdabad5-6512-4ee0-8a6e-a912f9a745ca)

**Comment by ANTIDARK** [comment]
预装了aria2的那个docker 镜像，怎么使用它预装的aria2？

**Comment by jianghaodn** [comment]
cao,我也是这个问题，然后看了这条才知道，原来是要选择一个目录进去再点下载来的。我之前是直接在主页点的离线下载，这样是不行的
